### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by Bupilipili**
### In this branch

-Added more styling in the application.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,13 +12,13 @@
 <body>
     <h1>Leaderboard</h1>
     <main class="content">
-    <article>
+    <article class="board">
    <div class="align">
         <h2 class="table-title">Recent scores</h2>
         <button type="button" class="refresh-button">Refresh</button>
```
Comment (shahadat3669) [inline src/index.html:18]: 👏🎉 Good job so far! Your project is coming along nicely. 🌟 However, there's a small improvement you can make to enhance the user experience.

- [x] [Optional]💡🔄 Currently, after clicking the submit button, the user needs to manually refresh the page to update the score list. But you can make it more user-friendly! 😊 After successfully adding the value to the API, you can call your `this.fetchScoresAndShowRecords()` function to automatically update the score list and display the new entry. This way, the user won't have to go through the extra step of refreshing the page.


🚀💻 By implementing this enhancement, you'll make your project even more seamless and user-friendly. Keep up the great work, and good luck with the rest of your project! You're doing fantastic! 💪🎉

File: src/index.html [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
     </div>
     <div class="table-section"></div>
```
Comment (shahadat3669) [inline src/index.html:20]: - [Optional]👍 Good job on displaying all the scores! However, there's a small improvement that can make the display even better. It's essential to show the top score at the top of the list to make it more user-friendly. To achieve this, I recommend sorting the scores in descending order before rendering them. By sorting the scores, the highest score will always be displayed at the top of the list, providing a better user experience and making it easier for users to find the top score. Keep up the great work, and happy coding! 🚀💻

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @Bupilipili , 

Great job so far! 👏 You've done well, and you're almost there for the next project ⏭️, but there are a few issues 🐛 that you still need to work on.




## Don't Worry 🙏

It's okay to have bugs in your code. 🆗 Please see this code review as an opportunity to learn practical coding skills and discover new tips that can improve your code quality 💯 and help you **get even better** 💼.

**I will guide you on how to address these areas down below 👇.**

## Highlights
✅ No linter Errors
✅ Descriptive PR Title
✅ Descriptive PR Description


## Required Changes ♻️

- [x] 👍 Thank you for the well-written README file! It provides a clear overview of your project and its functionalities. However, there's one important aspect that's missing: 
  - [x] instructions on how to run the app with the API(s) it consumes. To improve the README, I recommend adding details in the "Getting Started" section that includes detailed instructions on how to generate the keys or credentials for the API(s), how to set and store them securely, and how to incorporate them into your project. Since you are not sharing your API key, it's crucial to guide users on how to obtain their own API key and integrate it into the app.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Bupilipili) [reply src/index.html:18]: Thanks for the review, even thou it's optional but the project requirements needs data to be summoned from the API only when the refresh button is clicked.

File: src/index.html (hunk 2 above)
Comment (Bupilipili) [reply src/index.html:20]: Yes, am definitely gonna work on this. Thanks for the highlight.

**Review by DammyShittu — APPROVED** [review]
Hi @Bupilipili,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**